cmd/earthly/flag/global.go

```diff
@@ -101,6 +101,7 @@ type Global struct {
 	LocalSkipDB                string
 	DisableRemoteRegistryProxy bool
 	NoAutoSkip                 bool
+	IsGithubAnnotations        bool
```

Comment:
nit: drop the `Is`, e.g.
```suggestion
	GithubAnnotations        bool
```

or `EnableGithubAnnotations`

Reply:
Done